CHANGELOG.md

```diff
@@ -0,0 +1,34 @@
+openfortivpn
+============
+
+------
```

Comment:
In pure markdown, these "above lines" are not needed, they result in extra `<hr>` tags:
![capture d ecran de 2018-11-14 10-11-00](https://user-images.githubusercontent.com/5244945/48472115-cd2bbe80-e7f5-11e8-975b-70619c05a125.png)


You can just write:

```md
Legend
------
```